books/american_government/bake

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env ruby
+
+# frozen_string_literal: true
+
+require 'bundler/inline'
+
+gemfile do
+  gem 'openstax_kitchen', ENV['USE_LOCAL_KITCHEN'] ? { path: '/code/kitchen' } : '2.0.0'
+  gem 'slop', '4.8.2'
+  gem 'byebug'
+end
+
+recipe = Kitchen::BookRecipe.new(book_short_name: :american_gov) do |doc|
+  include Kitchen::Directions
+
+  book = doc.book
+
+  # Some stuff just goes away
+  book.search('cnx-pi').trash
+
+  BakePreface.v1(book: book, title_element: 'div')
+  BakeUnitTitle.v1(book: book)
+  BakeChapterTitle.v1(book: book)
+  BakeChapterIntroductions.v1(book: book)
+  BakeSuggestedReading.v1(book: book)
+
+  book.chapters.each do |chapter|
+    BakeChapterGlossary.v1(chapter: chapter, metadata_source: book.metadata)
+    BakeChapterSummary.v1(chapter: chapter, metadata_source: book.metadata)
+    BakeNonIntroductionPages.v1(chapter: chapter)
+
+    chapter.search('.review-questions, .critical-thinking').exercises.each do |exercise|
+      BakeNumberedExercise.v1(exercise: exercise, number: exercise.count_in(:chapter))
+    end
+
+    MoveExercisesToEOC.v1(
+      chapter: chapter,
+      metadata_source: book.metadata,
+      klass: 'review-questions'
+    )
+    MoveExercisesToEOC.v1(
+      chapter: chapter,
+      metadata_source: book.metadata,
+      klass: 'critical-thinking'
+    )
+
+    chapter.figures.each do |figure|
+      BakeFigure.v1(figure: figure,
+                    number: "#{chapter.count_in(:book)}.#{figure.count_in(:chapter)}")
+    end
+
+    chapter.tables('$:not(.unnumbered)').each do |table|
+      BakeNumberedTable.v1(table: table,
+                           number: "#{chapter.count_in(:book)}.#{table.count_in(:chapter)}")
+    end
+  end
+
+  book.pages('$.appendix').each do |page|
+    appendix_letter = [*('A'..'Z')][page.count_in(:book) - 1]
+
+    page.figures.each do |figure|
+      BakeFigure.v1(figure: figure, number: "#{appendix_letter}#{figure.count_in(:page)}")
+    end
+
+    page.tables('$:not(.unnumbered)').each do |table|
+      BakeNumberedTable.v1(table: table, number: "#{appendix_letter}#{table.count_in(:page)}", always_caption: true)
+    end
+
+    page.examples.each do |example|
+      BakeExample.v1(example: example,
+                     number: "#{appendix_letter}#{example.count_in(:page)}",
+                     title_tag: 'div')
+    end
+
+    BakeAppendix.v1(page: page, number: appendix_letter)
+  end
+
+  # Bake titled and unclassified notes
+  book.chapters.pages.notes.each do |note|
+    note.classes.empty? && BakeUnclassifiedNote.v1(note: note)
```

Comment:
Generally speaking I'm not a big fan of this kind of use of `&&`.  Would rather change

```ruby
condition && thing_to_do
```

to

```ruby
thing_to_do if condition
```

reads better IMO